.gitmodules

```diff
@@ -18,3 +18,6 @@
 [submodule "third_party/tock"]
 	path = third_party/tock
 	url = https://github.com/tock/tock.git
+[submodule "third_party/manticore"]
```

Comment:
Nitpick: Please add a blank line before the manticore submodule definition to be consistent with the rest of this file.